source/upgrading/upgrade/upgrade-4.17.rst

```diff
@@ -47,6 +48,13 @@ Overview of Upgrade Steps:
 
 .. include:: _customisation_warnings.rst
 
+.. warning::
+    If you are not already using the |sysvm64-version| System VM template you will need to
```

Comment:
@rohityadavcloud 
Is the new systemvm templates bundled with cloudstack-management package ?

Reply:
Yes, the new 4.17.2 systemvmtemplate is bundled with cloudstack-management package @weizhouapache 
(this boilerplate is same for all other files. I copied it from upgrade-4.16.rst)

Reply:
@rohityadavcloud 
thanks. if so, this part may be not required.
(systemvm registration is required for 4.16, but not since 4.17)

Reply:
Here's what we say in the doc build page @weizhouapache : https://qa.cloudstack.cloud/builds/docs-build/pr/295/upgrading/upgrade/upgrade-4.17.html

And this is same for all the upgrade 4.x.y versions:
<img width="761" alt="Screenshot 2022-12-13 at 2 36 33 PM" src="https://user-images.githubusercontent.com/95203/207274623-f167eede-6e07-4396-bc6b-ea4845548be2.png">

Reply:
@rohityadavcloud 
that's fine.

I meant the line 39 and the new warning (line 51 to line 54) can be removed, 
or change to other word like "**Upload the 4.17.2 System VM template if they are not bundled with cloudstack-management package.**"

now it looks like (it should be optional)
![image](https://user-images.githubusercontent.com/57355700/207284970-27062d80-c7ff-4cee-b006-a8ad3baad166.png)
and
![image](https://user-images.githubusercontent.com/57355700/207285205-16335dbb-a917-4f06-bdac-b4ecf8412f30.png)